src/helpers.sh

```diff
@@ -36,7 +36,7 @@ function helper::check_duplicate_functions() {
   local duplicates
   duplicates=$(echo "$sorted_names" | uniq -d)
   if [ -n "$duplicates" ]; then
-    state::set_duplicated_test_functions_found
+    state::set_duplicated_functions_merged "$script" "$duplicates"
```

Comment:
I just wanted to make it clear that this is a merged function, since the term duplicated function was already there by default, and I added two more global variables with similar names. Of course the modified version is also fine to me. Also, I noticed that the original code use the term 'duplicate functions' in some cases and 'duplicated functions'  in others, but I think their meaning is intended to be the same. It might be worthwhile to modify and standardize this as well.